platforms/hyperledger-fabric/configuration/roles/create/users/tasks/save_to_vault.yaml

```diff
@@ -0,0 +1,37 @@
+# Check user msp already created
+- name: Check if user msp already created
+  shell: |
+    vault kv get -field=admincerts {{ vault.secret_path | default('secretsv2') }}/crypto/peerOrganizations/{{ component_name }}/users/{{ user.identity }}/msp
+  environment:
+    VAULT_ADDR: "{{ vault.url }}"
+    VAULT_TOKEN: "{{ vault.root_token }}"
+  register: vault_user_result
```

Comment:
Ideally, we should also copy newly generated certificates (for the same key) in case of a refresh. The generate user crypto script has been modified to refresh an enrolment as needed. For example, a certificate gets refreshed if it is about to expire in next X hours, a certificate gets refreshed if for a given user the attributes change.